### PR TITLE
Update Ansible documentation to reflect change in Jaeger addon

### PR DESCRIPTION
**Pull request by geoand**
Needed since https://github.com/istio/istio/pull/3603 was merged

### CODE REVIEW

**Comment by sdake** [comment]
Weird - my filter didn't catch this review request into my reviews folder.  Apologies for delay in review.

File: _docs/setup/kubernetes/ansible-install.md [hunk 1]
```diff
@@ -57,8 +57,7 @@ The full list of configurable parameters is as follows:
 | `istio.dest` | The directory of the target machine where Istio will be installed | `~/.istio` (default) |
 | `istio.auth` | Boolean value to install Istio using MUTUAL_TLS | `true` and `false` (default) |
 | `istio.namespace` | The namespace where Istio will be installed | `istio-system` (default) |
-| `istio.addon` | Which Istio addons should be installed as well | This field is an array field, which by default contains `grafana`, `prometheus`, `zipkin` and `servicegraph` |
-| `istio.jaeger` | Whether or not Jaeger tracing should also be installed | `true` and `false` (default)|
+| `istio.addon` | Which Istio addons should be installed as well | This field is an array field, which by default contains `grafana`, `prometheus`, `zipkin`, `jaeger` (disables Zipkin if selected) and `servicegraph` |
```
Comment (sdake) [inline _docs/setup/kubernetes/ansible-install.md]: oxford comma would help clarity here e.g. "), and" although not a big deal.  Over time as you clean up this documentation might be something to consider for the future.

File: _docs/setup/kubernetes/ansible-install.md [hunk 2]
```diff
@@ -101,12 +100,12 @@ ansible-playbook main.yml -e '{"cluster_flavour": "k8s", "cmd_path": "~/kubectl"
...
 - User wants to install Istio on Openshift with settings other than the default
 ```bash
-ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "jaeger": true, "delete_resources": true}}'
+ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "delete_resources": true}}'
 ```
 
 - User wants to install Istio on Openshift but with custom add-on settings
 ```bash
-ansible-playbook main.yml -e '{"istio": {"delete_resources": true, "addon": ["grafana", "prometheus"]}}'
+ansible-playbook main.yml -e '{"istio": {"delete_resources": true, "addon": ["grafana", "prometheus", "jaeger"]}}'
```
Comment (sdake) [inline _docs/setup/kubernetes/ansible-install.md]: I saw somewhere prometheus was planned to be moved out of addons into the base install.  Might keep that in mind for the future (or I could be imagining that was planned).

**Comment by geoand** [comment]
No problem!

File: _docs/setup/kubernetes/ansible-install.md [hunk 3]
```diff
@@ -101,12 +100,12 @@ ansible-playbook main.yml -e '{"cluster_flavour": "k8s", "cmd_path": "~/kubectl"
 
 - User wants to install Istio on Openshift with settings other than the default
 ```bash
-ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "jaeger": true, "delete_resources": true}}'
+ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "delete_resources": true}}'
```
Comment (sdake) [inline _docs/setup/kubernetes/ansible-install.md]: this document could probably overall use some rework around the versions given that we are on 0.7.0, but imo looks good enough for merge.

**Comment by geoand** [comment]
Should I add anything specific to future PR titles to make it easier for
your filters to catch them?


Thanks

**Comment by sdake** [comment]
@geoand no - problem on my end :)  Thanks though for the offer.

**Comment by geoand** [comment]
Thank you for all the reviews 😀

**Comment by sdake** [comment]
@geoand note github is stating the branch is out of date and needs a master merge.

**Comment by geoand** [comment]
I'll take care of it in a few hours when I have access to my computer 😀

File: _docs/setup/kubernetes/ansible-install.md (hunk 1 above)
Comment (geoand) [reply _docs/setup/kubernetes/ansible-install.md]: Fixed :)

File: _docs/setup/kubernetes/ansible-install.md (hunk 2 above)
Comment (geoand) [reply _docs/setup/kubernetes/ansible-install.md]: Very interesting to know! When/if that happens I'll update the playbook to install Prometheus in a similar fashion as Jaeger is installed

File: _docs/setup/kubernetes/ansible-install.md (hunk 3 above)
Comment (geoand) [reply _docs/setup/kubernetes/ansible-install.md]: I updated the example to use `0.6.0`

**Comment by geoand** [comment]
@sdake The PR was rebased on the latest master and updated.

Thanks!

**Comment by sdake** [comment]
@geoand perhaps this changed in the intervening 8 hours, hwoever:

screen shot 2018-03-22 at 8 36 17 am

To the right of that should be a button "Merge to master" or something similar.  Can you try pressing that button?

TIA
-steve

**Comment by geoand** [comment]
@sdake Hopefully it's OK now :)

**Comment by sdake** [comment]
@geoand I don't have credentials to merge in this repo - perhaps @geeknoid can merge.

Cheers
-steve

**Comment by geoand** [comment]
Thanks a lot everyone!

**Pull request merged**